### PR TITLE
Made component responsive

### PR DESCRIPTION
**Pull request by boudewijnbout**

### CODE REVIEW

File: src/lib/components/StandardSectionButton.svelte [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	button {
+		background: none;
+		border: none;
+		padding: 1rem;
+		font-weight: 600;
+		font-size: 1rem;
+		cursor: pointer;
+	}
+
+	/* button:focus {
```
Comment (DaanKorver) [inline src/lib/components/StandardSectionButton.svelte:19]: Remove please

File: src/lib/styles/global.css [hunk 2]
```diff
@@ -20,7 +20,7 @@ img {
 }
 
 body {
-	background-color: var(--color-cmd-yellow);
+	/* background-color: var(--color-cmd-yellow); */
```
Comment (DaanKorver) [inline src/lib/styles/global.css:23]: remove

File: src/stores.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store'
+
+export const alert = writable('Welcome to the to-do list app!')
```
Comment (DaanKorver) [inline src/stores.js:3]: ??

**Review by DaanKorver — CHANGES_REQUESTED** [review]

File: src/lib/components/StandardSectionButton.svelte (hunk 1 above)
Comment (boudewijnbout) [reply src/lib/components/StandardSectionButton.svelte:19]: Removed.

File: src/lib/styles/global.css (hunk 2 above)
Comment (boudewijnbout) [reply src/lib/styles/global.css:23]: Fixed.

File: src/stores.js (hunk 3 above)
Comment (boudewijnbout) [reply src/stores.js:3]: Removed

**Review by DaanKorver — APPROVED** [review]

**Pull request merged**